### DISCUSSION TITLE
Seperate extension for vaultwarden??

### DISCUSSION THREAD

**Discussion by cyberqueenmeg** [dani-garcia/vaultwarden#3807] [Ideas]
Hi all,

Just wondering if there are plans for a seperate, not Bitwarden-affiliated, extension for vaultwarden. If not, I think it would be a cool idea.

**Comment by tessus** [comment]
The whole idea of vaultwarden is to be a compatible API server so that the clients do not have to be maintained. While the server is an important component, the most work is done in the clients (encryption/decryption, URL matching, attachment handling, ...).

Maintaining seperate clients is not the goal or in the scope of the vaultwarden project. 

Otherwise it would be a new product and maintaining the clients is certainly a lot more involved than working on the server component. If you look at how many components and resources are required to run a self-hosted Bitwarden instance, you will soon understand how nice it is to have a single binary instead.
